exercises/dnd-character/src/main/kotlin/DndCharacter.kt

```diff
@@ -0,0 +1,22 @@
+class DndCharacter {
+
+    val strength: Int = -1
+    val dexterity: Int = -1
+    val constitution: Int = -1
+    val intelligence: Int = -1
+    val wisdom: Int = -1
+    val charisma: Int = -1
+    val hitpoints: Int = -1
```

Comment:
`val smth = -1` means *forever*, that can confuse a student.
 Add *TODO*-notes, please :)